package.json

```diff
@@ -1,12 +1,13 @@
 {
 	"scripts": {
 		"test": "xo && ava && npm run build",
-		"build": "rollup -c",
-		"watch": "run-p --continue-on-error --silent \"build -- --watch\"",
-		"watch:sourcemap": "cross-env SOURCEMAP=true npm run watch --silent",
+		"build": "webpack",
+		"watch": "webpack --watch",
+		"watch:sourcemap": "webpack --watch --devtool eval-cheap-module-source-map",
+		"reduce-size": "babel --out-dir . --no-comments extension/*.js",
```

Comment:
It just wasn't applied :(

It might be a babel or babel-loader bug. That'd be its right place